### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by HansZizold**
** CHANGES **
Unit tests written for JS calculator files.
Jest snapshot tests are written for all the components.
User interaction simulated using React Library Tests for adding and multiplying operations.
Readme file updated

** CONTRIBUTORS **
All tests were developed in pair programming modality with the participation of:
[@memelopez](https://github.com/memelopez/)
[@HansZizold](https://github.com/HansZizold)

### CODE REVIEW

**Comment by memelopez** [comment]
we worked together in hans's project 👍🏼

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 
 ## History of Changes
 
+2022-07-13: Unit tests added using React Testing Library.
 2022-07-12: Math Magician application completed using react routes.
 2022-07-07: Calculator components refactored using hooks.
 2022-07-06: Calculator libraries and events handling added to provide calculator functionalities.
```
Comment (zieeco) [inline README.md]: - Please, kindly remove this section as it is not needed in the readme following the microverse [readme template](https://github.com/microverseinc/readme-template).
- Please, kindly update the live demo link of the app as it not taking users to the online version. You may choose to update the link or remove it totally to improve the user experience and to make your readme more professional.

Comment (zieeco) [reply README.md]: - Please, kindly update the "contributing" header by adding its missing content. you can refer to the  [readme template](https://github.com/microverseinc/readme-template) and ensure the link to the issue page of the calculator project is added to it.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great work so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Test are passing :fire: 


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @HansZizold, @memelopez and @zieeco 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Github inline comments feature is used properly.
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**